src/getStoreHandlers.js

```diff
@@ -1,5 +1,7 @@
 function format (string) {
+  /*eslint-disable no-unused-vars*/
   const [ _, action, state ] = `${ string }`.match(/(\w*)\_\d+\_(\w*)/, ' ') || []
+  /*eslint-enable no-unused-vars*/
```

Comment:
This rule is a bummer. So often we only use a second argument, particularly for general event publishers. I wish it could make an exception for vars/arguments that start with `_var`, like in Erlang.